.github/workflows/run-tests.sh

```diff
@@ -43,5 +43,56 @@ if [ $RUNTIME = "core" ] || [ $RUNTIME = "knative" ]; then
 
 elif [ $RUNTIME = "streaming" ]; then
   echo "TODO: test the streaming runtime"
+  # TODO move setup to fats
+  ${FATS_DIR}/install.sh helm
+  source ${FATS_DIR}/macros/helm-init.sh
+
+  helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+  helm install --name my-kafka incubator/kafka --set replicas=1,zookeeper.replicaCount=1,zookeeper.env.ZK_HEAP_SIZE=128m --namespace $NAMESPACE --wait
+
+  riff streaming kafka-provider create franz --bootstrap-servers my-kafka:9092 --namespace $NAMESPACE
+
+  cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dev-utils
+spec:
+  containers:
+  - name: dev-utils
+    image: projectriff/dev-utils
+EOF
+  kubectl create clusterrolebinding dev-util-stream --clusterrole=riff-streaming-readonly-role --serviceaccount=default:default
+
+  for test in java java-boot; do
+    name=system-streaming-cluster-repeater-${test}
+    image=$(fats_image_repo ${name})
+
+    riff function create $name --image $image --namespace $NAMESPACE --tail \
+          --git-repo https://github.com/${FATS_REPO}.git --git-revision ${FATS_REFSPEC} --sub-path functions/repeater/${test} &
+
+    riff streaming stream create letters --namespace $NAMESPACE --provider franz-kafka-provisioner --content-type 'text/plain'
```

Comment:
The value of `--provider` is generated, it needs to be looked up from the cluster at runtime